cocos/gfx/webgl2/webgl2-state-cache.ts

```diff
@@ -56,7 +58,10 @@ export class WebGL2StateCache {
     }
 
     initialize (texUnit: number, bufferBindings: number, vertexAttributes: number): void {
-        for (let i = 0; i < texUnit; ++i) this.glTexUnits.push({ glTexture: null });
+        for (let i = 0; i < texUnit; ++i) {
+            const glTexUnit: IWebGL2TexUnit = { glTexture: null };
+            this.glTexUnits.push(glTexUnit);
```

Comment:
Why do it like this?

Reply:
Because it's difficult to infer the interface type by 
```ts
this.glTexUnits.push({ glTexture: null });
```

`{glTexture: null}` is a plain object with typeinfo.
By declaring its type `const glTexUnit: IWebGL2TexUnit = { glTexture: null };` explicitly, we could easily get its type and mangle `glTexture` property.

Reply:
Got it. But it also define a new variable `glTexUnit`. How much package size will it save?

Reply:
glTexUnit is just an object reference, and it cost really little, no new objects will be created.